maintaining.rst

```diff
@@ -0,0 +1,276 @@
+
+.. _Backporting:
+
+Backporting
+===========
+
+Collection maintainers backport merged pull requests to stable branches
+following the `semantic versioning <https://semver.org/>`_ and release policies of the collections.
+
+For more information about the process, refer to the `Backporting guidelines <https://docs.ansible.com/ansible/devel/community/development_process.html#backporting-merged-prs-in-ansible-core>`_.
+
+For convenience, backporting can be implemented automatically using GitHub bots (for example, with the `Patchback app <https://github.com/apps/patchback>`_) and labeling like it is done in `community.general <https://github.com/ansible-collections/community.general>`_ and `community.network <https://github.com/ansible-collections/community.network>`_.
+
+.. _Releasing:
+
+Releasing
+=========
+
+Collection maintainers release all supported stable versions of the collections regularly,
+provided that there have been enough changes merged to release.
+
+Generally, releasing in the collections consists of:
+
+  1. Planning and announcement.
+  2. Generating a changelog.
+  3. Creating a release git tag and pushing it.
+  4. Automatic publishing the release tarball on `Ansible Galaxy <https://galaxy.ansible.com/>`_ by Zuul.
+  5. Final announcement.
+
+For more information about the process, refer to the `Releasing guidelines <releasing.rst>`_.
+
+.. _Expanding community:
+
+Expanding community
+===================
+
+Increasing a number of active contributors and maintainers
+----------------------------------------------------------
+
+Maintainers are interested in increasing a number of active long-term contributors for a collection they maintain.
+
+Contributors are reviewers, issue or pull request authors, testers, maintainer, and all other people who help develop the project.
+
+Every regular contributor once was a newcomer. Make the first experience as positive as possible to make the new people coming back.
+
+Good development documentation makes contributors life much easier. Get feedback from new contributors if there were things they struggled with when working on their proposals and improve the documentation correspondingly.
+
+Create the ``CONTRIBUTING`` file in your repository. In there, add a link to the `Quick-start guide <create_pr_quick_start_guide.rst>` as well as to other guidelines describing things specific to your collection.
+
+Make contributors feel welcome. Greet and thank contributors impersonally in ``README`` and individually in their proposals.
+Thank all participants after merging or closing a proposal.
+
+Be responsive. Respond as quickly as possible. Even if you cannot review a proposal right now, greet and thank the author.
+
+Add and keep updated the ``CONTRIBUTORS`` file listing all the contributors including issue reporters and refer to it from your ``README``.
```

Comment:
I'm not really sure about this, as keeping an eye on this for every new PR and new contributor would become hard.

Reply:
done